support-frontend/assets/helpers/abTests/__tests__/abtestTest.ts

```diff
@@ -21,840 +18,303 @@ jest.mock('ophan', () => ({
 }));
 
 // ----- Tests ----- //
-const emptySettings = {
-	switches: {
-		experiments: {},
-	},
-	amounts: {
-		GBPCountries: {},
-		UnitedStates: {},
-	},
-};
-
-const acquisitionDataMockTestControl = {
-	componentId: 'acquisitions-subscription-banner',
-	componentType: 'ACQUISITIONS_SUBSCRIPTIONS_BANNER',
-	abTest: {
-		name: 'mockTest',
-		variant: 'control',
-	},
-};
-
-const acquisitionDataMockTestVariant = {
-	componentId: 'acquisitions-subscription-banner',
-	componentType: 'ACQUISITIONS_SUBSCRIPTIONS_BANNER',
-	abTest: {
-		name: 'mockTest',
-		variant: 'variant',
-	},
-};
-
-const acquisitionDataMockTest2Control = {
-	componentId: 'acquisitions-subscription-banner',
-	componentType: 'ACQUISITIONS_SUBSCRIPTIONS_BANNER',
-	abTest: {
-		name: 'mockTest2',
-		variant: 'control',
-	},
-};
-
-const acquisitionDataMockTest2Variant = {
-	componentId: 'acquisitions-subscription-banner',
-	componentType: 'ACQUISITIONS_SUBSCRIPTIONS_BANNER',
-	abTest: {
-		name: 'mockTest2',
-		variant: 'variant',
-	},
-};
-
-const mockTestControl = `/test.html?acquisitionData=${encodeURI(
-	JSON.stringify(acquisitionDataMockTestControl),
-)}`;
-
-const mockTestVariant = `/test.html?acquisitionData=${encodeURI(
-	JSON.stringify(acquisitionDataMockTestVariant),
-)}`;
-
-const mockTest2Control = `/test.html?acquisitionData=${encodeURI(
-	JSON.stringify(acquisitionDataMockTest2Control),
-)}`;
-
-const mockTest2Variant = `/test.html?acquisitionData=${encodeURI(
-	JSON.stringify(acquisitionDataMockTest2Variant),
-)}`;
-
-describe('basic behaviour of init', () => {
+
+describe('init', () => {
```

Comment:
I made quite a lot of changes here! The main philosophy was simplifying the test code and removing redundant tests. I've tried to remove as much noise as possible from the test cases so that it's easier to see at a glance the important parts that we're actually testing